src/WindowsAzureMessaging/WindowsAzureMessaging/SBConnectionString.m

```diff
@@ -7,61 +7,48 @@
 
 @implementation SBConnectionString
 
-NSString* const defaultListenSasRuleName = @"DefaultListenSharedAccessSignature";
-NSString* const defaultFullSasRuleName = @"DefaultFullSharedAccessSignature";
-
-
-+ (NSString*) stringWithEndpoint:(NSURL*)endpoint issuer:(NSString*) issuer issuerSecret:(NSString*)secret
-{
-    if(!endpoint || !issuer || !secret)
-    {
-        NSLog(@"endpoint/issuer/secret can't be null.");
-        return nil;
-    }
-    
-    endpoint = [SBNotificationHubHelper modifyEndpoint:endpoint scheme:@"sb"];
-    
-    NSString* endpointUri = [endpoint absoluteString];
-    if([[endpointUri lowercaseString] hasPrefix:@"endpoint="])
-    {
-        return [NSString stringWithFormat:@"%@;SharedSecretIssuer=%@;SharedSecretValue=%@",endpointUri,issuer,secret];
-    }
-    else
-    {
-        return [NSString stringWithFormat:@"Endpoint=%@;SharedSecretIssuer=%@;SharedSecretValue=%@",endpointUri,issuer,secret];
-    }
+NSString *const defaultListenSasRuleName = @"DefaultListenSharedAccessSignature";
+NSString *const defaultFullSasRuleName = @"DefaultFullSharedAccessSignature";
+
++ (NSString *)stringWithEndpoint:(NSURL *)endpoint issuer:(NSString *)issuer issuerSecret:(NSString *)secret {
+  if (!endpoint || !issuer || !secret) {
```

Comment:
The two space indent makes Objective-C much harder to read. I don't think this change belongs in this particular PR. We need to have more discussion/consensus on the proper formatting rules.

Reply:
Fixed formatting rules in the clang format to use 4 spaces.